src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import userReducer from "./user/userReducer";
+
+const store = createStore(userReducer, applyMiddleware(thunk));
```

Comment:
I know here you only have one reducer but to make your codebase future proof you should have created `rootReducer` and passed that value here.